service/src/main/java/tds/assessment/web/endpoints/AssessmentController.java

```diff
@@ -46,13 +48,18 @@ public AssessmentController(final AssessmentService service) {
      */
     @GetMapping(value = "/{clientName}/assessments/{key}", produces = MediaType.APPLICATION_JSON_VALUE)
     @ResponseBody
-    ResponseEntity<Assessment> findAssessment(@PathVariable final String clientName, @PathVariable final String key)
-        throws NotFoundException {
+    ResponseEntity<Assessment> findAssessment(@PathVariable final String clientName,
+                                              @PathVariable final String key) throws NotFoundException {
         final Assessment assessment = service.findAssessment(clientName, key)
             .orElseThrow(() -> new NotFoundException("Could not find set of admin subject for %s", key));
 
         return ResponseEntity.ok(assessment);
     }
 
-
+    @DeleteMapping(value = "/{clientName}/assessments/{key}", produces = MediaType.APPLICATION_JSON_VALUE)
+    ResponseEntity<?> removeAssessment(@PathVariable final String clientName,
+                                       @PathVariable final String key) throws NotFoundException {
+        service.removeAssessment(clientName, key);
```

Comment:
What happens if you try to call this endpoint with:

```
/SBAC_PT/assessments/(SELECT _key FROM itembank.tblsetofadminsubjects limit 1)
```

Not sure what the encoding would be or how Spring would handle that.  I'm only asking because we're doing straight String building for SQL queries which can lead to SQL injection attacks.  Maybe this isn't a concern since this is contained within kubernetes and never exposed.

Reply:
I did consider that. Not sure if Spring would process it, but even if it did we are doing a check at the service layer to see if the the assessment exists - so at worst it would return a 404.